src/components/ContentPreview/ContentPreview.js

```diff
@@ -46,7 +46,9 @@ type Props = {
     isSmall: boolean,
     autoFocus: boolean,
     useHotkeys: boolean,
-    showSidebar?: boolean,
+    contentSidebarProps: {
```

Comment:
Why is this a mandatory field? Can it not be a defaultProp {}?

Reply:
nope!